apps/cowswap-frontend/src/modules/swap/hooks/useSwapDerivedState.ts

```diff
@@ -16,19 +16,21 @@ export function useSwapDerivedState(): SwapDerivedState {
 export function useFillSwapDerivedState(): void {
   const isProviderNetworkUnsupported = useIsProviderNetworkUnsupported()
   const updateDerivedState = useSetAtom(swapDerivedStateAtom)
+  const rawState = useAtomValue(swapRawStateAtom)
   const derivedState = useBuildTradeDerivedState(swapRawStateAtom, true)
 
   const slippage = useTradeSlippage()
 
   useEffect(() => {
     updateDerivedState(
       isProviderNetworkUnsupported
-        ? DEFAULT_TRADE_DERIVED_STATE
+        ? { ...DEFAULT_TRADE_DERIVED_STATE, isUnlocked: false }
```

Comment:
To be consistent with other trading widgets, it's better to do in `swapDerivedStateAtom`:
```
export const swapDerivedStateAtom = atom<SwapDerivedState>({
  ...DEFAULT_TRADE_DERIVED_STATE,
  isUnlocked: false
})
```